eva-server/src/main/java/uk/ac/ebi/eva/server/configuration/EvaPropertyConfiguration.java

```diff
@@ -0,0 +1,9 @@
+package uk.ac.ebi.eva.server.configuration;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackages = {"uk.ac.ebi.eva.lib.config"})
```

Comment:
The main `Application` class should scan this package. That would make the component scan a single, centralized step.